### ISSUE TITLE
[Bug]: Extra white space at the bottom of pdf

### ISSUE THREAD

**Issue by OlegDO** [puppeteer/puppeteer#10612]
### Minimal, reproducible example

```TypeScript
Html:
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Red div</title>
</head>
<body>
  <div style="height: 340px; width: 1025px; background: red">Red div</div>
</body>
</html>

Code:
const browser = await puppeteer.launch({
      headless: 'new',
      args: ['--no-sandbox', '--disable-setuid-sandbox', '--disable-gpu'],
    });
const page = await this.browser.newPage();

await page.setContent(htmlString);

const pdf = await page.pdf({
      printBackground: true,
      pageRanges: '1',
      height: '340px',
      width: '1025px',
    });
```

### Error string

no error

### Bug behavior

- [ ] Flaky
- [X] PDF

### Background

Generated pdf contain extra white space at the bottom of the pdf

PDF:
[puppeteer-html-to-pdf.pdf](https://github.com/puppeteer/puppeteer/files/12122550/puppeteer-html-to-pdf.pdf)

Screen of this pdf: 
<img width="1389" alt="image" src="https://github.com/puppeteer/puppeteer/assets/44732463/2cc31661-4a0a-4a8e-acde-e36ca91e29a9">

### Expectation

Generated pdf should not contain extra white space at the bottom

### Reality

Generated pdf contain extra white space at the bottom

### Puppeteer version

20.9.0

### Node version

16.10.0

### Package manager

npm

### Package manager version

7.24.0

### Operating system

macOS

**Comment by OrKoN** [comment]
would you mind re-posting this to crbug.com? I remember seeing a similar issue in Chromium but I could not find it anymore. Please post a link to crbug.com here.

**Comment by OrKoN** [comment]
I think I have found the issue https://bugs.chromium.org/p/chromium/issues/detail?id=1396392

**Comment by OrKoN** [comment]
The earliest patcheset has been published in 117.0.5847.0 so we will need to wait for that to become available via Puppeteer or you could test using a Canary/Dev build.

**Comment by Choco-milk-for-u** [comment]
work pretty much fine for me.
![image](https://github.com/puppeteer/puppeteer/assets/84465375/9be48aac-5997-4c0b-aee3-703d6d35a676)
```js
const puppeteer = require("puppeteer");
const path = require("path");
(async () => {
  const browser = await puppeteer.launch({
    headless: "new",
    args: ["--no-sandbox", "--disable-setuid-sandbox", "--disable-gpu"],
  });
  const page = await browser.newPage();

  await page.setContent(`<!DOCTYPE html>
  <html lang="en">
    <head>
      <title>Red div</title>
    </head>
    <body>
      <div style="height: 340px; width: 1025px; background: red">Red div</div>
    </body>
  </html>
  `);

  const pdf = await page.pdf({
    printBackground: true,
    pageRanges: "1",
    height: "340px",
    width: "1025px",
    path: path.join(__dirname, "/try.pdf"),
  });
})();

```
